Stratis.Bitcoin/Features/MemoryPool/MempoolFeature.cs

```diff
@@ -8,15 +8,57 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Transaction memory pool feature for the Full Node.
+    /// <see cref="https://github.com/bitcoin/bitcoin/blob/6dbcc74a0e0a7d45d20b03bb4eb41a027397a21d/src/txmempool.cpp"/>
```

Comment:
`<see>` is used anywhere as inline reference and the tag is replaced by the referenced name in the text.

Example:

```
<summary>
This method compares <see cref="Apples" /> and <see cref="Oranges" />
</summary>
```

Intellisense will give you "This method compares Apples and Oranges." 
And in MSVS you are able to hit F12 on "Apples" (and "Oranges" as well) to go to those classes.

`<see>` can also be used as follows:

```
Bla bla bla bla - see `<see cref="..." /> for more information.
```

whereas `<seealso>` creates "see also" section as can be seen e.g. here https://msdn.microsoft.com/en-us/library/hh229816.aspx
which is a separate section at the end of the documentation page (if we generated such pages, which we can)